site/content/docs/5.2/forms/select.md

```diff
@@ -61,6 +75,13 @@ As is the `size` attribute:
 
 Add the `disabled` boolean attribute on a select to give it a grayed out appearance and remove pointer events.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This disabled `<select>` element can not be used because it does no respect the Orange Design System specifications.
```

Comment:
Indeed, designers said that the actual look of this disabled variant is OK and that they will change the DSM accordingly => OK for removing this callout.

Reply:
**TO DO:**
ask design if input of 50px height are allowed. 
**If yes**, add example of that in DSM + delete callout message

**If no**, the new message should be:
This select variant, with labels that are not above input fields, should not be used because it does not respect the Orange Design System specifications.

For the correct design, please refer to [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) and [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) on the Orange Design System website.